core/src/test/scala/org/apache/spark/memory/StaticMemoryManagerSuite.scala

```diff
@@ -159,13 +156,13 @@ class StaticMemoryManagerSuite extends MemoryManagerSuite {
     assert(mm.acquireUnrollMemory(dummyBlock, 500L, evictedBlocks))
     // `spark.storage.unrollFraction` is 0.4, so the max unroll space is 400 bytes.
     // Since we already occupy 60 bytes, we will try to ensure only 400 - 60 = 340 bytes.
```

Comment:
actually, this whole test should be rewritten to trigger the case where unroll evicts part of existing storage. I have some changes here https://github.com/andrewor14/spark/commit/78cd2be5efd14dcad4fcd648f6815d741c09694d that are passing tests. I'll submit a PR against your branch shortly.